src/rp2/locales/kl/LC_MESSAGES/messages.po

```diff
@@ -630,6 +630,48 @@ msgstr "__test_Yearly Gain / Loss Summary"
 msgid "Yellow"
 msgstr "__test_Yellow"
 
+msgid "airdrop"
+msgstr ""
```

Comment:
This should be `__test_airdrop`. Same for the ones below: they all start with `__test_` followed by the English word.